docs/root/version_history/current.rst

```diff
@@ -36,6 +36,9 @@ Changes
   tracing is not forced.
 * router: allow retries of streaming or incomplete requests. This removes stat `rq_retry_skipped_request_not_complete`.
 * router: allow retries by default when upstream responds with :ref:`x-envoy-overloaded <config_http_filters_router_x-envoy-overloaded_set>`.
+* router: internal redirect configs are moved to the :ref`internal_redirect_policy
+  <envoy_api_field_router.RouterAction.internal_redirect_policy>` field, which defines more fine
+  grained control of the internal redirect behavior.
```

Comment:
If we keep a functional difference w.r.t  default behavior with schemes, we should document it here.

Reply:
Done.